src/app/artist-details/artist-details.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="card2" *ngIf="artist">
+  <div class="card-header2">
+    <div>
+      <h1>
+        <span label label-default style="align-content: center;">ARTIST PROFILE</span>
+      </h1>
+    </div>
+  </div>
+
+  <div class="card-body">
+    <div class="block-21 mb-4 d-flex" *ngIf="artist">
+      <a class="blog-img mr-4"></a>
+      <img class="center-block img-responsive" [style.width.px]="200" [style.margin.px]="2"
```

Comment:
The style attributes here can be moved into a CSS file rather, because they do not need to be applied dynamically.

Reply:
@strizzwald fixed